### PR TITLE
Add info.json file

### PR DESCRIPTION
**Pull request by XVhc6A**
### Brief description of the changes:
We should have an info file and keep the version updated so it's easier to track down bugs.

### Tested Changes:
- [ ] I've tested the changes locally or with people.
- [ ] I've not tested the changes.

### CODE REVIEW

**Comment by TheBigZet** [comment]
Why am I listed as the author? :D

**Comment by XVhc6A** [comment]
who do you think should be author @TheBigZet ? Since you're the one that manages it I think it's fair to call you 'author' since it's really just who manages the mod as it's deployed to the factorio portal, not necessarily who wrote it.

File: info.json [hunk 1]
```diff
@@ -1 +1,10 @@
-null
+{
+  "name": "Biter Battles - Community Edition",
+  "version": "5.0.0",
+  "title": "Biter Battles CE",
+  "author": "TheBigZet",
```
Comment (TheBigZet) [inline info.json:5]:
```suggestion

```

**Review by TheBigZet — CHANGES_REQUESTED** [review]

**Comment by Ragnarok77-factorio** [comment]
The discord link is dead, would probably remove it, and probably rename it to free bb instead..? asking on discord what people think

**Pull request merged**